src/js/lib.js

```diff
@@ -304,7 +301,7 @@ var isEmpty = function(obj) {
  * @private
  */
 var hasClass = function(element, classToCheck){
-  return ((' ' + element.className + ' ').indexOf(' ' + classToCheck + ' ') !== -1);
+  return ((` ${element.className} `).indexOf(` ${classToCheck} `) !== -1);
```

Comment:
The spaces here have a specific purpose that I think gets blurred when you add them in a template like this.